docgen/src/guide/Server-side_rendering.md

```diff
@@ -6,8 +6,24 @@ category: guide
 navWeight: 30
 ---
 
-React InstantSearch is not yet compatible with server-side rendering. This is one of our priority topics
-right now. We will soon provide API entries to handle this usecase.
+React InstantSearch builds the Algolia query based on every widget declared. That's why at the first rendering there are no results yet. This is inconvenient when doing server side rendering as the html generated by the server will not contain any hits. 
+
+If you want to have a server side rendering application using React InstantSearch you can use the special API entries we provide for this particular use-case: 
+
+* a new type of `<InstantSearch>` is available under `'react-instantsearch/server'`
+* a `findResults` function is available under `'react-instantsearch/server'`
+
+Let's see how it works!
+
+First you will need to get the first results using the `findResults` function. This function takes two parameters: 
```

Comment:
Either show a very small example or directly link to the example, otherwise this level of detail at this place is too much and I am not yet ready to read this/understand this.